frontend/src/components/sidebar.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import './sidebar.css'
+import Link from 'next/link'
+import Image from 'next/image'
```

Comment:
I am curious why you are importing <Image/> when you use <img/> to display the logo.

Reply:
This was from left over from previous implementation, Ill update